pkgs/by-name/mo/mongodb-ce/package.nix

```diff
@@ -12,6 +12,7 @@
   nix-update,
   gitMinimal,
   pup,
+  nixosTests,
```

Comment:
Don't use `nixosTests` for checking version, use `versionCheckHook`.

Example: https://github.com/NixOS/nixpkgs/blob/79ade21b8e26f875deb621de8f34a494e2e51bef/pkgs/by-name/ty/typst/package.nix#L10

Reply:
I didn't touch the version check, I just added the newly introduced nixosTest (for ensuring compatibility with the mongodb NixOS module) to the passthru tests

Reply:
Nonetheless, I migrated the package to use the install version check instead now: 57f5fc2df901f138634932ffbf6672eb8a690acc

Reply:
Thank you, it's perfect.